tools/cli/adminDBScanCommand.go

```diff
@@ -141,11 +147,12 @@ type (
 	// ProgressReport contains metadata about the scan for all shards which have been finished
 	// This is periodically printed to stdout
 	ProgressReport struct {
-		NumberOfShardsFinished           int
+		NumberOfShardsFinished           int64
```

Comment:
`int64` seems like an overkill for # of shards.

Reply:
Agree but I will need to cast to int64 anyways below.